components/_global-navigation.scss

```diff
@@ -75,6 +119,12 @@ $nav-width: 1024px;
 			width: 80px;
 		}
 
+		&.wds-has-multiple-subitems {
```

Comment:
My first idea was the same as yours, but I couldn't center the text inside the `fixed-width` containers. If you have some tips or just know how to handle it—it would be appreciated.